packages/types/lib/connection/api/get.ts

```diff
@@ -84,6 +84,7 @@ export type PostPublicConnection = Endpoint<{
             | Omit<BasicApiCredentials, 'raw'>
             | Omit<TbaCredentials, 'raw'>
             | { type: 'APP'; app_id: string; installation_id: string }
+            | { type: 'CUSTOM'; app_id: string; installation_id: string }
```

Comment:
our credentials management is so weird. Why is the access token part of the connection config and not credentials? (I'm just realising it's the same issue with APP)